src/linker/specifications/environment.yaml

```diff
@@ -1,5 +1,5 @@
 computing_environment: slurm
-# container_engine: singularity  # singularity or docker; comment out completely if unknown
+container_engine: singularity  # or docker
```

Comment:
Did you mean to change this? It seems like how it was would still be correct.

Reply:
It was intentional, but I see your point. I think my intent was to make it the practical default of always specifying, but that maybe goes too far.